Makefile

```diff
@@ -15,9 +15,13 @@ VM_TYPE=hvm
 # For route53.tf
 PRIVATE_DOMAIN=$(CLUSTER_NAME).local
 
+# Development cluster variables, required by worker_dev module
+DEV_APP_FROM_PORT:= 8080
+DEV_APP_TO_PORT:= 8081
```

Comment:
@hazim1093 which port is this? user can deploy 1 to M applications to the DEV cluster